lib/version.rb

```diff
@@ -0,0 +1,3 @@
+module RavenDB
+  VERSION = "4.1.0-alpha".freeze
```

Comment:
I think it's going to be `4.0.0-rc1`. @ppekrol Do you agree?

Reply:
Right. I was testing bundle generation and changed that. Fixed and pushed.